data/xml/N19.xml

```diff
@@ -4,9 +4,18 @@
     <meta>
       <booktitle>Proceedings of the 2019 Conference of the North <fixed-case>A</fixed-case>merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)</booktitle>
       <url>N19-1</url>
-      <editor><first>Jill</first><last>Burstein</last></editor>
-      <editor><first>Christy</first><last>Doran</last></editor>
-      <editor><first>Thamar</first><last>Solorio</last></editor>
+      <editor>
+        <first>Jill</first>
+        <last>Burstein</last>
+      </editor>
+      <editor>
+        <first>Christy</first>
+        <last>Doran</last>
+      </editor>
+      <editor>
+        <first>Thamar</first>
+        <last>Solorio</last>
+      </editor>
       <publisher>Association for Computational Linguistics</publisher>
```

Comment:
I'm not sure how you created this, but it broke our formatting, which puts `<author>` and `<editor>` tags one per line. See the [indent function](https://github.com/acl-org/acl-anthology/blob/master/bin/anthology/utils.py) which should fix this for you.

Reply:
Thanks for the push! Still not quite right. `git diff` should only show changes on the `<video>` lines....

Reply:
You're absolutely correct and thanks for pointing that out! The issues should be fixed and the only differences should be the video lines.

For the future, I'll see what I can do to resolve that issue in my code that does the adding of the videos to the XML to make sure the formatting is correct.

Reply:
Can you add your script under `bin/` so others can use it in the future? How does it relate to Arne's implementation?